src/test/components/MenuButtons.test.js

```diff
@@ -41,9 +45,24 @@ jest.mock('../../utils/gz');
 import { shortenUrl } from '../../utils/shorten-url';
 jest.mock('../../utils/shorten-url');
 
+import { symbolicateProfile } from 'firefox-profiler/profile-logic/symbolication';
+jest.mock('firefox-profiler/profile-logic/symbolication');
+
 // Mock hash
 const hash = 'c5e53f9ab6aecef926d4be68c84f2de550e2ac2f';
 
+// We need IndexedDB to get a SymbolStore that's necessary for symbolication
+// to even start, in some of the tests for this file.
+function resetIndexedDb() {
```

Comment:
Question: Would it be better to put this in a re-usable utility so other tests can have access to it in the future?

Reply:
Yeah, that could be a good idea.